packages/api/src/index.ts

```diff
@@ -1,404 +1,817 @@
-import axios, { AxiosRequestConfig, AxiosResponse, AxiosError } from 'axios'
-import { AdxUri, resolveName } from '@adxp/common'
-import * as auth from '@adxp/auth'
-import { AdxSchemas, AdxRecordValidator, AdxViewValidator } from '@adxp/schemas'
-import * as t from './types'
-import * as err from './errors'
-import * as ht from './http-types'
-
-export type QP = Record<string, any> | URLSearchParams
-
-export enum PdsEndpoint {
-  GetDid,
-  Account,
-  Session,
-  Repo,
-  RepoCollection,
-  RepoRecord,
-  View,
-}
+/**
+* GENERATED CODE - DO NOT MODIFY
+* Created Tue Sep 20 2022
+*/
+import {
+  Client as XrpcClient,
+  ServiceClient as XrpcServiceClient,
+} from '@adxp/xrpc'
+import { methodSchemas, recordSchemas } from './schemas'
```

Comment:
looks like `recordSchemas` isn't used here

Reply:
Yeah that's a todo